index.bs

```diff
@@ -240,6 +240,11 @@ An event-level trigger configuration is a [=struct=] with the following items:
 
 </dl>
 
+<h3 id="aggregation-coordinator-header">Aggregation coordinator</h3>
+
+An <dfn>aggregation coordinator</dfn> is one of a user-agent-determined [=set=]
```

Comment:
nit: s/user-agent-determined/user agent-determined/

Reply:
We use `user-agent-determined` in public key section, so will leave as is for consistency.